ahrefs/monorobot
# Ignore code comments from defined list of users

## Description of the task

Closes #121 
Allows for configuring ignored users on PRs.

## How to test

Run the test suite

```
make test
```

Or specific payload:
```
./_build/default/src/monorobot.exe check_gh mock_payloads/issue_comment.created_in_issue_by_ignored.json
```
NB: make sure your ignore list have: "ignored_user" in there.
